stripe/__init__.py

```diff
@@ -39,6 +39,7 @@
     Customer,
     Dispute,
     Event,
+    EphemeralKey,
```

Comment:
Sorry, this is the pedantic part of me, but mind flipping the two "E"s here so that it's fully alphabetized? Thanks!